client/flutter/flutter-netlify-build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
- Please create a new directory `helper-scripts` (or something similar)
- Add a short readme to the directory saying something like 

> This directory contains helper for scripts for building, releasing & ci